randnum.c

```diff
@@ -0,0 +1,22 @@
+#include <stdio.h>
```

Comment:
Great code - you figured out the efficient way of implementing this program! Just again, comments!